lib/preprocess.js

```diff
@@ -11,10 +12,12 @@ export const preprocess = async ({
   const measurements = (await fetchMeasurements(web3Storage, cid))
     .filter(measurement => {
       try {
+        // eslint-disable-next-line camelcase
+        measurement.participantAddress = parseParticipantAddress(measurement.wallet_address)
```

Comment:
The callback of a `Array#filter` function shouldn't modify the data, otherwise the code is harder to reason about. Could you make it clearer by either mapping in an `Array#map` or refactoring this to a `for of` loop?

Reply:
> The callback of a `Array#filter` function shouldn't modify the data, otherwise the code is harder to reason about (one doesn't expect it).

Agreed, thanks for calling this out.

> Could you make it clearer by either mapping in an `Array#map` or refactoring this to a `for of` loop?

👍🏻

Reply:
Fixed in [7c976f7](https://github.com/filecoin-station/spark-evaluate/pull/10/commits/7c976f7e2e69cfaec821e2d45d6abe50fb255daa)